packages/input-date/src/LionInputDate.js

```diff
@@ -50,7 +50,7 @@ export class LionInputDate extends LocalizeMixin(LionInput) {
     // modelValue is localized, so we take the timezone offset in milliseconds and subtract it
     // before converting it to ISO string.
     const offset = modelValue.getTimezoneOffset() * 60000;
-    return new Date(modelValue - offset).toISOString().slice(0, 10);
+    return new Date(modelValue.getTime() + offset).toISOString().slice(0, 10);
```

Comment:
I believe for us. it should be `-`, since this change made one test fail.

I'll dive in it more deeply tomorrow.

Reply:
The problem is in the way dates get handled:

<img width="431" alt="Screenshot 2020-06-25 at 11 35 52" src="https://user-images.githubusercontent.com/10164421/85695373-441a8000-b6d8-11ea-913f-a9c2aa8684d4.png">

So for now all we need to do is change the value in the test to `new Date('2000/12/12')` for `input-date` & `input-datepicker`.

With your fix the `serializedValue` is correct (for negative timezones), but the `formattedValue` still displays wrong.

Reply:
it seems that `new Date('2000/12/12')` is equivalent to `new Date('2000-12-12T00:00:00')`

I can change the tests to 2000/12/12 and remove the "fix"

Reply:
I pushed new commit with this and removed the fix